pkg/autoscaler/autoscaler.go

```diff
@@ -70,21 +71,21 @@ func (r *Autoscaler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	return ctrl.Result{}, nil
 }
 
-func (a *Autoscaler) Start() {
-	for range time.Tick(a.Interval) {
-		if !a.LeaderElection.IsLeader.Load() {
+func (r *Autoscaler) Start() {
```

Comment:
Why rename to `r`?

Reply:
The name is `r` for the other methods. The linter highlighted the inconsistency.

Reply:
`a` seems to make more sense for `Autoscaler`. I think the others might have used `r` because they were reconcilsers?